sofa-boot-project/sofa-boot-core/runtime-sofa-boot/src/main/java/com/alipay/sofa/runtime/invoke/DynamicJvmServiceProxyFinder.java

```diff
@@ -112,6 +125,71 @@ public ServiceComponent findServiceComponent(ClassLoader clientClassloader, Cont
         return null;
     }
 
+    public void afterBizStartup(Biz biz) {
+        // Currently, there is no way to get SOFA Runtime Manager from biz
+        // The overhead is acceptable as this only happens after biz's successful installation
+        for (SofaRuntimeManager runtimeManager: SofaFramework.getRuntimeSet()) {
+            if (runtimeManager.getAppClassLoader().equals(biz.getBizClassLoader())) {
```

Comment:
biz.getBizClassLoader().equals(runtimeManager.getAppClassLoader()) instead, current biz must not null